Adding next step (#2768)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mailbox-migration/migrating-imap-mailboxes/migrating-your-outlook-com-account.md

9. You'll get a message that two-step verification is turned on. Print your new *recovery code* (this isn't your app password). If you ever need to recover, access to this account, this recovery code will help. It's a good idea to keep it tucked away in a safe place.

10. Choose **Next**.

> Once you finish turning on two-step verification and creating an app password, continue here [IMAP migration in the Microsoft 365 admin center](imap-migration-in-the-admin-center.md).

Reviewer: ```suggestion
Once you have turned on two-step verification and created an app password, continue with the [IMAP migration in the Microsoft 365 admin center](imap-migration-in-the-admin-center.md).
```

> Once you have turned on two-step verification and created an app password, continue with the [IMAP migration in the Microsoft 365 admin center](imap-migration-in-the-admin-center.md).

Reviewer 2: ```suggestion
After you turn on two-step verification and create an app password, continue with the [IMAP migration in the Microsoft 365 admin center](imap-migration-in-the-admin-center.md).
```
